moveit_ros/moveit_servo/config/xbox360_wired.yaml

```diff
@@ -0,0 +1,28 @@
+axis_linear:
```

Comment:
One of the questions I have when looking at this is:  Why are there 2 config files? It seems like `joystick_to_servo.yaml` is for joint commands, `xbox360_wired.yaml` is for Cartesian commands?